tests/sqlparser_common.rs

```diff
@@ -16533,6 +16537,74 @@ fn parse_create_procedure_with_parameter_modes() {
     }
 }
 
+#[test]
+fn create_procedure_with_parameter_default_value() {
+    let sql = r#"CREATE PROCEDURE test_proc (IN a INTEGER = 1, OUT b TEXT = '2', INOUT c TIMESTAMP = NULL, d BOOL = 0) AS BEGIN SELECT 1; END"#;
+    match verified_stmt(sql) {
+        Statement::CreateProcedure {
+            or_alter,
+            name,
+            params,
+            ..
+        } => {
+            assert_eq!(or_alter, false);
+            assert_eq!(name.to_string(), "test_proc");
+            let fake_span = Span {
+                start: Location { line: 0, column: 0 },
+                end: Location { line: 0, column: 0 },
+            };
+            assert_eq!(
+                params,
+                Some(vec![
+                    ProcedureParam {
+                        name: Ident {
+                            value: "a".into(),
+                            quote_style: None,
+                            span: fake_span,
+                        },
```

Comment:
```suggestion
                        name: Ident::new("a"),
```
I think this can be simplified? that would let us skip introducing the fake span as well

Reply:
Yes, certainly. I did it this way because that's how the similar modes test did it, but I will simplify